### PR TITLE
Added a section about the Normal scoped registry to the Get Started page

### PR DESCRIPTION
**Pull request by NormalMark**

### CODE REVIEW

File: docs/essentials/getting-started.md [hunk 1]
```diff
@@ -45,4 +47,13 @@ That’s it! You’re now up and running with Normcore. Now, I know this example
 - [XR Avatars and Voice Chat](../guides/xr-avatars-and-voice-chat.md)
 - [Synchronizing Custom Data](../realtime/synchronizing-custom-data.md)
 
-Until next time  : )
\ No newline at end of file
+Until next time  : )
+
+## Normal scoped registry
```
Comment (maxweisel) [inline docs/essentials/getting-started.md]: This is kind of awkward at the end of the Getting Started guide. It feels a little too advanced for beginners and makes the Getting Started guide feel less simple.

Will you make this into a dedicated page? What do you think is the best place for it? / Where should we put it on the sidebar?

File: docs/essentials/getting-started.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ Once you have the plugin imported and Unity has had a chance to recompile succes
 
 ![](./getting-started/package-manager.png "An empty project with Normcore freshly imported.")
 
+The plugin will have also added the [Normal scoped registry](#normal-scoped-registry) to the project.
```
Comment (maxweisel) [inline docs/essentials/getting-started.md]: This wording is a little awkward. Maybe "The plugin will automatically add the [Normal scoped registry] to the project." ?

Comment (maxweisel) [inline docs/essentials/getting-started.md]: This wording is a little awkward. Maybe "The plugin will automatically add the [Normal scoped registry] to the project." ?

**Review by maxweisel — CHANGES_REQUESTED** [review]
Added some quick notes

File: docs/essentials/getting-started.md (hunk 1 above)
Comment (NormalMark) [reply docs/essentials/getting-started.md]: I'm thinking:
* Page title: Normal Scoped Registry.
* Section: Essentials. However it's not actually essential and would clutter that section.
* Section: Architecture. It's not related to architecture either but at least it's out of the way.

Or even create a new section? It doesn't 100% fit the existing ones.

Comment (maxweisel) [reply docs/essentials/getting-started.md]: Architecture works for me for now.

**Review by maxweisel — APPROVED** [review]

**Pull request merged**